myreactapp/src/App.js

```diff
@@ -0,0 +1,77 @@
+
+import React from 'react';
+import './App.css';
+import Navbar from "./components/Navbar/Navbar";
+import Newfan from './Newfan'
+
+
+
+
+
+function App() {
+
+  return (
```

Comment:
it looks like your have two return functions. I would delete this one since it looks like a generic return.

Reply:
wait this looks like it has your nav bar in it... is this the newest return function?